src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -47,13 +47,13 @@ const MSG = defineMessages({
   rolesTooltip: {
     id: 'dashboard.ActionsPageFeed.ActionsPageEvent.rolesTooltip',
     defaultMessage: `{icon} {role, select,
-      0 {This permission allows users to put the colony in and out of 
+      0 {This permission allows users to put the colony in and out of
```

Comment:
VSCode decided that it should automatically remove some redundant spaces. I've checked the tool tips, and this change did not cause any side effects.